kolibri/core/analytics/pskolibri/__init__.py

```diff
@@ -0,0 +1,270 @@
+from __future__ import absolute_import
+from __future__ import division
+
+import os
+import sys
+import time
+
+from kolibri.core.analytics.pskolibri.common import memoize_when_activated
+from kolibri.core.analytics.pskolibri.common import NoSuchProcess
+from kolibri.core.analytics.pskolibri.common import PY3
+
+POSIX = os.name == "posix"
+WINDOWS = os.name == "nt"
+LINUX = sys.platform.startswith("linux")
+MACOS = sys.platform.startswith("darwin")
+_TOTAL_PHYMEM = None
+_timer = getattr(time, 'monotonic', time.time)
```

Comment:
There are a couple of these spread around -- maybe they could be maintained in `kolibri.utils.compat`? Slight tweaks for the names would be prefixing with `IS_`, like `IS_POSIX` etc...

Reply:
I haven't been able to find other constants or requests to find out the OS. I have added them all in the common.py file in pskolibri, so they are available for the whole application there.
kolibri.utils.compat seems to be focused in different versions of python compatibility, not in the OS.

Reply:
My comment doesn't appear next to the code that I originally wrote it for, but I think you responded to what I meant :)

The thing that I would think is that if we don't keep generic compatibility layers in a centralized place, we end up with a lot of modules re-inventing the same structures. So I thought that detecting the OS seems like a pretty generic thing that we could perhaps benefit from moving outside.

Reply:
But I understand that it's perhaps part of the original `psutil` and if we do other things than just deleting code, it gets a bit hard to maintain...

Reply:
One day, a new Linux kernel og Windows 12 will come and ask us some questions about what's going on here :)

So I guess I think it's fine to keep these constants together with their original code! :+1: